coremltools/converters/mil/mil/ops/defs/control_flow.py

```diff
@@ -499,6 +626,27 @@ def type_inference(self):
 
 @register_op(doc_str="")
 class list_scatter(Operation):
+    """
+    Scatter ``values`` to ``ls`` at locations ``indices``.
+    
+    Parameters
+    ----------
+    ls: List[*] (Required)
+    
+    indices: <num_updates,i32> (Required)
+        * Indices of ``ls`` to scatter to.
+        * Elements of ``indices`` must be in ``[0, ls.length)`` at runtime.
+    
```

Comment:
I guess we missed one of the parameters `value` in the documentation.

Reply:
Is this correct?

>     value: <*,T> (Optional)
>     	* Element value to write, which must match the element shape of ``ls``.
>     	* Default is ``None``.
>